Link to docs on the Foxglove website (#144)
foxglove/ros-foxglove-bridge: README.md

The `foxglove_bridge` uses the [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol), a similar protocol to rosbridge but with the ability to support additional schema formats such as ROS 2 `.msg` and ROS 2 `.idl`, parameters, graph introspection, and non-ROS systems. The bridge is written in C++ and designed for high performance with low overhead to minimize the impact to your robot stack.

## Installation and setup

For information on how to install and configure your `foxglove_bridge`, check out the full docs on the [Foxglove website](https://foxglove.dev/docs/studio/connection/using-foxglove-bridge).

## Build from source

> To build `foxglove_bridge` from source, you must first fetch your source and install dependencies:

Reviewer: ```suggestion
To build `foxglove_bridge` from source, you should clone the source to your ROS workspace and install dependencies:
```
